modules/govuk/lib/puppet/parser/functions/govuk_node_class.rb

```diff
@@ -21,6 +21,10 @@ module Puppet::Parser::Functions
       "govuk_node_class: Wrong number of arguments given #{args.size} for 0."
     ) unless args.size == 0
 
+    # Explicit early return if in AWS.
+    aws_migration = lookupvar('aws_migration')
+    return aws_migration unless (aws_migration.nil? or aws_migration.empty?)
```

Comment:
What about `||` instead of `or`? I'm never sure what the difference is, but I don't see `or` very often.